x/globalfee/types/params.go

```diff
@@ -45,6 +90,29 @@ func validateMinimumGasPrices(i interface{}) error {
 	return dec.Validate()
 }
 
+// todo check if correct?
+func validateBypassMinFeeMsgTypes(i interface{}) error {
+	_, ok := i.([]string)
```

Comment:
validate:
 []string{} ok
[]string{""} not ok
 []string{"abc"}, must be sdk msg types.